WebApplication/ClientApp/package.json

```diff
@@ -26,6 +26,9 @@
     "@hig/tooltip": "^2.1.0",
     "@hig/top-nav": "^4.1.0",
     "@hig/typography": "^2.1.0",
+    "@types/jest": "^27.4.0",
```

Comment:
Are there really any changes in this files ? I see only items moved around

Reply:
I didn't change **package.json** and **package-lock.json** manually. However I launched **npm** **install** and I trust what have happened is that **npm** automatically ordered packages to be in alphabetic order. Since I saw only benefit from dependencies being sorted I left it in a PR.

Reply:
@hradilm let me know please if I should revert the changes to the **package.json**

Reply:
yes please. It does not make sense to commit files without change,

Reply:
I wil re-open this PR as after reverting package.json it still showing like there were some changes in the file, though there are no changes